docs/src/main/asciidoc/security-overview-concept.adoc

```diff
@@ -2,264 +2,33 @@
-
-|===
-| | quarkus-oidc| quarkus-smallrye-jwt | quarkus-elytron-security-oauth2
-
-|Requires Bearer JWT verification
-|Local verification or introspection
-|Local verification
-|Introspection
-|Requires Bearer opaque token verification
-|Introspection
-|No
-|Introspection
-|Refreshing `JsonWebKey` set for verifying JWT tokens
-|Yes
-|Yes
-|No
-|Represent token as `Principal`
-|Yes
-|Yes
-|Yes
-|Inject JWT as MP JSON Web Token (JWT)
-|Yes
-|Yes
-|No
-|Authorization Code Flow
-|Yes
-|No
-|No
-|Multi-tenancy
-|Yes
-|No
-|No
-|User info support
-|Yes
-|No
-|No
-|PEM key format support
-|No
-|Yes
-|No
-|SecretKey support
-|No
-|In JSON Web Key (JWK) format
-|No
-|Inner-signed and encrypted or encrypted tokens
-|Introspection
-|Local verification
-|Introspection
-|Custom token verification
-|No
-|With injected JWT parser
-|No
-|Accept JWT as a cookie
-|No
-|Yes
-|No
-|===
-
-[[identity-providers]]
-== Identity providers
-
-
-The JPA `IdentityProvider` creates a `SecurityIdentity` instance, which is used during user authentication to verify and authorize access requests making your Quarkus application secure.
-
-
-`IdentityProvider` converts the authentication credentials provided by `HttpAuthenticationMechanism` to a `SecurityIdentity` instance.
-
-Some extensions, for example, `OIDC`, `OAuth2`, and `SmallRye JWT` have inline `IdentityProvider` implementations specific to the supported authentication flow.
-For example, `quarkus-oidc` uses its own `IdentityProvider` to convert a token to a `SecurityIdentity` instance.
-
-If you use `Basic` or `Form` HTTP-based authentication then you must add an `IdentityProvider` instance that can convert a username and password to a `SecurityIdentity` instance.
-
-* For more information about `Basic` or `Form` HTTP-based authentication, see the following resources:
-** xref:security-basic-authentication-tutorial.adoc[Secure a Quarkus application with Basic authentication]
-** xref:security-jdbc.adoc[Using security with JDBC]
-** xref:security-ldap.adoc[Using security with an LDAP realm]
-
-== Authorization
-
-Quarkus also supports role-based access control (RBAC).
-For more information about RBAC and other authorization options in Quarkus, see xref:security-authorization-of-web-endpoints-reference.adoc[Security authorization].
-
-== Quarkus Security customization
-
-Quarkus Security is highly customizable. You can customize the following core security components of Quarkus:
+The Quarkus Security framework is highly customizable and can be extended.
+You can customize the following core security components of the Quarkus Security:
 
 * `HttpAuthenticationMechanism`
 * `IdentityProvider`
 * `SecurityidentityAugmentor`
 
 For more information about customizing Quarkus Security including reactive security, and how to register a security provider, see xref:security-customization.adoc[Security customization].
 
-== Combining authentication mechanisms
```

Comment:
Note: All of this H2 section of security-overview-concept.adoc is now moved (as-is) to the end of a new concept topic in this PR named security-authentication-mechanisms-concept.adoc.